### ISSUE TITLE
Consider default of normalized=True for all STFT and Mel transforms

### ISSUE THREAD

**Issue by turian** [lucidrains/audiolm-pytorch#82]
`normalized = False` by default in torch STFTs and Mels. Consider using a default of `normalized = True`, particularly if computing multiple length transforms.

This is done in Encodec: https://github.com/facebookresearch/encodec/blob/main/encodec/msstftd.py#L62

And I had a non-audiolm model I trained recently that didn't work until I flipped this variable.

This won't completely solve the loss balancing issue #61 but might nonetheless improve training

**Comment by faroit** [comment]
@turian Interesting spot. Can you explain where this has helped making things work? Isn't this a matter of input scaling and standardization?

**Comment by turian** [comment]
Here are some more examples: https://github.com/search?q=normalized%3Dtrue+stft&type=code

I haven't really dug into this too much, but I say at the very least it's a program option @lucidrains and would be curious what people training up audiolm-pytorch find from it

**Comment by lucidrains** [comment]
@turian Hey Joseph! Glad you raised this issue as I've been looking at that and wasn't sure what the proper values are

I've made it so one can [configure this](https://github.com/lucidrains/audiolm-pytorch/commit/1831daed7059ed9150d6fbee1bf886c5276555a4), either on loading the audio data or during stft. Let me know if I missed a spot

Could you point to any literature that shows normalization is preferable?

**Comment by turian** [comment]
@lucidrains TBH it's hard to say. I've seen both things happen in code. Unfortunately, this is a detailed that tends to be elided in the literature. I like your approach of making it False by default, but encouraging people who are testing particularly the multiscale stuff to flip it on. I'll make a few comments on https://github.com/lucidrains/audiolm-pytorch/commit/1831daed7059ed9150d6fbee1bf886c5276555a4

**Comment by lucidrains** [comment]
@turian yes, i understand how this is

ok, we'll go with that strategy and let the crowd wisdom kick in

**Comment by turian** [comment]
Yeah. I do think the normalization is more important when you have multiple FFT sizes. Another example: 

https://github.com/acids-ircam/ddsp_pytorch/blob/1edc19e0e8d70a69c8083a58456f6894c1623baf/ddsp/core.py#L36

**Comment by turian** [comment]
And agreed that discovering the right parameters is an empirical question.
